website/content/cli/usage/overview/structure-and-navigation.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Structure and Navigation
```

Comment:
The videos in this page are not relevant and need to be removed. Generally, something you can read and reference is better for this type of material.